source/modules/collection-api/event/actions/create.md.erb

```diff
@@ -1,27 +1,34 @@
 ### Event Creation
 
-```php
-<?php
-
-use Inspetor;
-
-$inspetor = $this->getConfiguredInspetor();
-
-$inspetor->trackEventCreation($event);
-
-?>
+```sh
+curl -XPOST https://collection.inspcdn.net/events \
+    -d '{
+        "id":          "12345",
+        "name":        "Reveillon 2020",
+        "is_physical": True,
+        "event_datetimes":  [
+            {
+                "id":           1,
+                "event_datetime": 1574210000
```

Comment:
The docs for Event is defining `event_datetime` and `id` as `String`, but they're `int` here

Reply:
Good catch! I changed to `int` in the overview (since it's meant to be a Unix-timestamp, I think it's safe/good to make that imposition)